tidy3d/components/device/simulation.py

```diff
@@ -33,16 +35,25 @@
 
 from ...log import log
 
-HEAT_BACK_STRUCTURE_STR = "<<<HEAT_BACKGROUND_STRUCTURE>>>"
+DEVICE_BACK_STRUCTURE_STR = "<<<DEVICE_BACKGROUND_STRUCTURE>>>"
 
+HeatBCTypes = (TemperatureBC, HeatFluxBC, ConvectionBC)
+HeatSourceTypes = (UniformHeatSource, HeatSource, HeatFromElectricSource)
+ElectricBCTypes = (VoltageBC, CurrentBC, InsulatingBC)
 
-class HeatSimulation(AbstractSimulation):
-    """Contains all information about heat simulation.
+
+class DeviceSimulationType(str, Enum):
+    HEAT = "HEAT"
+    CONDUCTION = "CONDUCTION"
+
+
+class DeviceSimulation(AbstractSimulation):
+    """Contains all information about device simulations.
```

Comment:
I think the docstring needs a thorough explanation of what a DeviceSimulation is, what it does, and how It compares to Simulation. including links to all the docs that mention it. See the Simulation docstring